x-pack/plugin/deprecation/src/main/java/org/elasticsearch/xpack/deprecation/MlDeprecationChecker.java

```diff
@@ -67,22 +69,23 @@
     }
 
     static Optional<DeprecationIssue> checkModelSnapshot(ModelSnapshot modelSnapshot) {
-        if (modelSnapshot.getMinVersion().before(Version.V_7_0_0)) {
+        if (modelSnapshot.getMinVersion().before(MIN_CHECKED_SUPPORTED_SNAPSHOT_VERSION)) {
             StringBuilder details = new StringBuilder(
                 String.format(
                     Locale.ROOT,
-                    "model snapshot [%s] for job [%s] supports minimum version [%s] and needs to be at least [%s].",
+                    // Important: the Kibana upgrade assistant expects this to match the pattern /[Mm]odel snapshot/
+                    // and if it doesn't then the expected "Fix" button won't appear for this deprecation.
+                    "Model snapshot [%s] for job [%s] has an obsolete minimum version [%s].",
```

Comment:
@masseyke and @debadair I've added one word compared to what you used in #79387: "Model snapshot" instead of just "Snapshot" at the beginning.  Please let me know if you strongly object to this.